### ISSUE TITLE
libgfortran-4.dll is linked, but Julia only ships with libgfortran-3.dll for now

### ISSUE THREAD

**Issue by Gnimuc** [JuliaPackaging/BinaryBuilder.jl#159]
```julia
julia> versioninfo()
Julia Version 0.6.2
Commit d386e40c17* (2017-12-13 18:08 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-6267U CPU @ 2.90GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, skylake)

julia> hdl = Libdl.dlopen("liblbfgsb-3.dll")
ERROR: could not load library "liblbfgsb-3.dll"
The specified module could not be found.

Stacktrace:
 [1] dlopen(::String, ::UInt32) at .\libdl.jl:97 (repeats 2 times)
```

<img width="1088" alt="dep" src="https://user-images.githubusercontent.com/7437173/36362293-3883a2ca-156e-11e8-9c9f-1b332e89130b.png">

how could I select libgfortran's version? or is there a simple workaround?

**Comment by staticfloat** [comment]
The workaround is that once https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/161 is merged, a new rootfs is going to be shipped that will use `GCC 6.4.0` when targeting Windows computers, which will link against `libgfortran-3.dll`.

**Comment by Gnimuc** [comment]
Thanks! ❤️
